lib/routes/bflai/announcements.ts

```diff
@@ -0,0 +1,193 @@
+import { Route, DataItem, Data } from '@/types'; // RSSHub 类型定义
+import ofetch from '@/utils/ofetch'; // HTTP 请求库
+import { load } from 'cheerio'; // HTML 解析
+import { parseDate } from '@/utils/parse-date'; // 日期解析
+import cache from '@/utils/cache'; // 缓存工具
+
+const ROOT_URL = 'https://bfl.ai'; // 将根URL定义为常量，方便维护
+
+/**
+ * 辅助函数：获取并解析单个公告详情页，提取正文内容。
+ * 使用缓存机制 (cache.tryGet) 避免重复请求。
+ *
+ * @param {string} itemLink - 文章详情页的完整链接。
+ * @param {string} fallbackDescription - 如果详情页抓取失败或内容为空时使用的备用描述 (通常是列表页的摘要)。
+ * @returns {Promise<string>} 文章的描述内容 (HTML格式)。
+ */
+const fetchDescription = (itemLink: string, fallbackDescription: string): Promise<string> =>
+    cache.tryGet(itemLink, async () => {
+        // console.log(`   CACHE MISS - Fetching detail page: ${itemLink}`); // 调试时可保留注释，方便以后维护者调试
+        let detailPageHtml;
+        try {
+            detailPageHtml = await ofetch(itemLink, {
+                headers: {
+                    'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.0.0 Safari/537.36 RSSHub',
```

Comment:
Why use this UA? Does the site only works with this fixed UA string over RSSHub's [randomised UA](https://github.com/DIYgod/RSSHub/blob/ef22a71dd60ae8936204e26c3ed0ccce450e26a5/lib/config.ts#L464)?